test/envtest/dlbdeviceplugin_controller_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envtest
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	devicepluginv1 "github.com/intel/intel-device-plugins-for-kubernetes/pkg/apis/deviceplugin/v1"
+)
+
+var _ = Describe("DlbDevicePlugin Controller", func() {
+
+	const timeout = time.Second * 30
+	const interval = time.Second * 1
+
+	AfterEach(func() {
+		time.Sleep(time.Second * 2)
+
+	})
+
+	Context("Basic CRUD operations", func() {
+		It("should handle DlbDevicePlugin objects correctly", func() {
+			spec := devicepluginv1.DlbDevicePluginSpec{
+				Image:        "testimage",
+				NodeSelector: map[string]string{"feature.node.kubernetes.io/dlb": "true"},
```

Comment:
we probably don't need to set NodeSelector here as it will be part of plugin yaml when your other PR is merged.

Reply:
Umm,,, sorry, I don't understand two things.

1. Isn't what I did for [this matter](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/760#discussion_r755996314)? I thought I understood what you meant but now I am confused.!

2. How is this related to another PR I made..? I thought this is more related to deployment/operator/samples/yaml file than the plugin yaml file and the operator?

Reply:
1. your link points to the top of this PR, so I didn't get what you mean
2. We need help from Mikko on this I guess. I thought we're testing controller code in envtests.

Reply:
1. Sorry. I didn't know the link does not go to there if it's resolved and closed. What I wanted to ask is "Was what I put for satisfying what you suggested?"

2. I see.! @mythi :) Please help us (maybe on Monday)!

Reply:
I believe @hj-johannes-lee refers to [this comment](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/760/files#r755996314).

None of our `envtest` cases check the resulting daemonset is as expected. We should start adding those checks we don't need them in this PR for DLB. This PR is OK as it is now.

Reply:
@mythi
yeap! it is what I wanted to refer.!
Then, can I regard this comment as resolved without changing anything?